src/strategies/web3subgraph/transformer.mjs

```diff
@@ -3,13 +3,13 @@ const version = "0.1.0";
 
 import { toJSON } from "../../disc.mjs";
 
-function generate(address, tokenId) {
+function generate(address, tokenId, blockNumber) {
   if (address === "0x01ab7d30525e4f3010af27a003180463a6c811a6") {
     return {
       type: "extraction",
       version: "0.0.1",
       name: "soundxyz",
-      args: [tokenId],
+      args: [tokenId, `0x${blockNumber.toString(16)}`],
```

Comment:
`eth-fun` is available as a dependency in main rn, let's please use https://github.com/rugpullindex/eth-fun/blob/master/API.md#tohexnumber